core/src/Kokkos_HIP_Space.hpp

```diff
@@ -715,6 +699,8 @@ class HIP {
   static int concurrency();
   static const char* name();
 
+  hipStream_t hip_stream() const;
```

Comment:
Where is that function implemented?  Same function for the new constructor.

Reply:
These changes sneaked in the wrong commit.